src/content/en/2021/structured-data.md

```diff
@@ -17,4 +19,568 @@ featured_stat_3: TODO
+
+We can also see a <a hreflang="en" href="https://www.w3.org/TR/skos-primer/">SKOS</a> property—`skos:concept`—on 0.04% of pages. SKOS is another standard, which aims to provide a way of describing taxonomies and classifications (e.g., tags, data sets, and so on).
+
+#### Dublin Core
+
+<a hreflang="en" href="https://dublincore.org/">Dublin Core</a> is a vocabulary interoperable with linked data standards that was originally conceived in Dublin, Ohio in 1995 at an OCLC (Online Computer Library Center) and NCSA (National Center for Supercomputing Applications) workshop.
+
+It was designed to describe a broad range of resources (both digital and physical) and can be used in various business scenarios. Starting in 2000 it became extremely popular among RDF-based vocabularies and received the adoption of the W3C.
+
+Since 2008 it is managed by the Dublin Core Metadata Initiative (DCMI) and remains highly interoperable with other linked data vocabularies. It is typically implemented as a collection of meta tags in a HTML document.
+
+{{ figure_markup(
+  image="structured-data-dublin-core.jpg",
+  caption="Dublin Core usage",
+  description="Bar chart showing the usage of Dublin Core tags across the web. Of all mobile pages in our data: `dc.title` is on 0.70%, `dc.language` is on 0.49%, `dc.description` is on 0.44%, `dc.publisher` is on 0.22%, `dc.creator` is on 0.21%, `dc.subject` is on 0.20%, `dc.source` is on 0.19%, `dc.identifier` is on 0.17%, `dc.relation` is on 0.16%, `dcterms.title` is on 0.15%, `dc.type` is on 0.14%, `dcterms.rightsholder` is on 0.12%, and `dcterms.identifier` is on 0.11%. Desktop usage is similar.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3kZ1Ys-9tId-WBa_8muMzrTAu1Ad5TXYgkopXMmBVc1xmd2N_4PZIpEZEOqRL7baymle0kHzaC6KY/pubchart?oid=1985156978&format=interactive",
+  sheets_gid="358057466",
+  width=600,
+  height=574,
+  sql_file="dublin_core_types.sql"
+  )
+}}
+
+That the most popular attribute type is `dc:title` (on 0.70% of pages) comes as no surprise; but it is interesting to see that `dc:language` is next (above common descriptors like _description_, _subject_ and _publisher_) with a penetration of 0.49%. This makes sense, when you consider that Dublin Core is often used in multilingual metadata management systems.
+
+It's also interesting to see the relatively prominent appearance of `dc:relation` (on 0.16% of pages)—an attribute that is capable of expressing relationships between different concepts.
+
+While it might seem to many that Schema.org is predominant in the context of SEO, the role of DC remains pivotal because of its broad interpretation of concepts and its deep roots in the _linked open data movement_.
+
+#### Social metadata
+
+##### Open Graph
```

Comment:
Headings should have at least one sentence and not be empty (so called "Stacked headings", which are used to group headings but don't have any content themselves). "A heading is like a chapter title; it must have at least a sentence of information below it. Stacked headings can indicate inefficient organization of information."